### ISSUE TITLE
struct.error while using opencv-viewer.py

### ISSUE THREAD

**Issue by matze946** [bitcraze/aideck-gap8-examples#70]
Hey :)
I tried to use the opencv-viewer.py  and got a struct.error message 

`Connecting to socket on 192.168.4.1:5000...
Socket connected
Traceback (most recent call last):
  File "streamer.py", line 86, in <module>
    [magic, width, height, depth, format, size] = struct.unpack('<BHHBBI', imgHeader)
struct.error: unpack requires a buffer of 11 bytes`

Any ideas how to solve this?

**Comment by knmcguire** [comment]
Hi! Did you update your wifi example with the latest example? or are you trying it out right out of the box? You would probably need to revert the examples to this commit: bdcca78839a0e61baee97e610d9c73b91d9934e8 in that case.

**Comment by matze946** [comment]
Yes I first used it without updating to the latest example. But after updating the examples the error is still the same..

**Comment by evoggy** [comment]
I haven't seen this before. Do you get this the first time you connect as well? Or is it only after disconnecting/connecting again?

**Comment by kjayarajah** [comment]
Anybody have any luck with this? I'm having this issue even after going back to the older commit. Thanks in advance for any help -- been stuck with this for several days.

**Comment by matze946** [comment]
I updated everything again like in the getting started pages [https://www.bitcraze.io/documentation/tutorials/getting-started-with-aideck/]
That solved the problem for me, but now I am still not able to stream the images with the opencv-viewer.py 
It says that I am connected to the socket, but I can't see the viewer.
Any advice on that issue?

**Comment by knmcguire** [comment]
Good you fixed that build problem.

Were you able to flash over air the wifi example okay without error? Are you connected to the aodeck as accesspoint or is it connecting to wifi itself?

**Comment by matze946** [comment]
Yes I was able to flash the wifi example using the radio without any errors and I'm connected with the aideck as accesspoint.

**Comment by knmcguire** [comment]
Ah oke. Sometimes I had to powercycle my crazyflie a time or two before it was connected. You can also have feedback from the console tab of the Cfclient for any cpx readouts. Usually it would say if it manage to establish a connection or if anything failed. If the camera failed to initialized for instance, you'll see that in the console tab.

**Comment by knmcguire** [comment]
ah great! we are trying to work on the stability of it all, as it should not be necessary to constantly restart it... but I'll keep it open here so that other people know of it if they are seeing the same thing of what they should do.
